### PR TITLE
Add Apache load balancer doc

### PR DESCRIPTION
**Pull request by doxiao**

### CODE REVIEW

File: site/apache.md [hunk 1]
```diff
@@ -0,0 +1,460 @@
+
+# Load Balancing with Apache Web Server
+
+This page describes how to setup and start a Apache Web Server for load balancing inside a Kubernets cluster. The configuration and startup can either be automatic when you create a domain using the WebLogic Operator's `create-weblogic-domain.sh` script, or manually if you have an existing WebLogic domain configuration.
+
+## Build Docker Image for Apache Web Server
+
+You need to prepare the Docker image for Apache Web Server that enbeds Oracle WebLogic Server Proxy Plugin.
```
Comment (mriccell) [inline site/apache.md]: You need to build the

File: site/apache.md [hunk 2]
```diff
@@ -0,0 +1,460 @@
...
+
+```
+
+# Load balancer to deploy.  Supported values are: VOYAGER, TRAEFIK, APACHE, NONE
+
+loadBalancer: APACHE
+
+```
+
+The `create-weblogic-domain.sh` script installs the Apache Web Server with Oracle WebLogic Server Proxy Plugin into the Kubernetes *cluster*  in the same namespace of the *domain*.
```
Comment (mriccell) [inline site/apache.md]: in the same namespace as the domain

File: site/apache.md [hunk 3]
```diff
@@ -0,0 +1,460 @@
...
+
+The Apache Web Server will expose a `NodePort` that allows access to the load balancer from the outside of the Kubernetes cluster.  The port is controlled by the following setting in `create-weblogic-domain-inputs.yaml` file:
```
Comment (mriccell) [inline site/apache.md]: allows access to the load balancer from outside of the Kubernetes cluster.
The port is configured by setting 'loadBalancerWebPort'  in `create-weblogic-domain-inputs.yaml` file.

File: site/apache.md [hunk 4]
```diff
@@ -0,0 +1,460 @@
...
+
+domain1-apache-webtier                                    2h
+
+```
+
+
+### Use your own plugin WL module configuration
+
+
+Optionally you can fine turn the behavior of the Apache plugin by providing your own Apache plugin configuration for `create-weblogic-domain.sh` script to use. You put your custom_mod_wl_apache.conf file in a local directory, say `/scratch/apache/config` , and specify this location in the `create-weblogic-domain-inputs.yaml` file as follows.
```
Comment (mriccell) [inline site/apache.md]: You can fine turn the behavior of the Apache plugin by providing your own Apache plugin configuration, provide your own script that will replace  `create-weblogic-domain.sh` script.
Replace "say" for "for example.
Do not provide /scratch/apache/config directory give a generic example this confuses users, also it should be aligned with the image example for creating the apache image

File: site/apache.md [hunk 5]
```diff
@@ -0,0 +1,460 @@
...
+
+```
+
+# Docker volume path for APACHE
+
+# By default, the VolumePath is empty, which will cause the volume mount be disabled
+
+loadBalancerVolumePath: /scratch/apache/config
```
Comment (mriccell) [inline site/apache.md]: Use generic config path do not use scratch

File: site/apache.md [hunk 6]
```diff
@@ -0,0 +1,460 @@
...
+     $ docker tag 12213-apache:latest store/oracle/apache:12.2.1.3
+
+```
+
+More information about the Apache plugin can be found at: [Apache Web Server with Oracle WebLogic Server Proxy Plugin on Docker](https://docs.oracle.com/middleware/1213/webtier/develop-plugin/apache.htm#PLGWL395).
+
+Once you have access to the Docker image of the Apache Web Server, you can go ahead follow the instructions below to setup and start Kubernetes artifacts for Apache Web Server.
+
+
+## When use Apache load balancer with a WebLogic domain created with the WebLogic Operator
```
Comment (rosemarymarano) [inline site/apache.md]: Should this title be, "When to use the Apache load balancer with a WebLogic domain created with the WebLogic Operator"?

File: site/apache.md [hunk 7]
```diff
@@ -0,0 +1,460 @@
...
+
+# Docker volume path for APACHE
+
+# By default, the VolumePath is empty, which will cause the volume mount be disabled
+
+loadBalancerVolumePath: /scratch/apache/config
+
+```
+
+Once the loadBalancerVolumePath is specified, the `create-weblogic-domain.sh` script will use the custom_mod_wl_apache.config file in `/scratch/apache/config` directory to replace what is in the Docker image.
```
Comment (mriccell) [inline site/apache.md]: Use generic config path do not use scratch

File: site/apache.md [hunk 8]
```diff
@@ -0,0 +1,460 @@
...
+
+The generated yaml files will look similar except that the lines that start with "#" will be un-commented like the following.
```
Comment (mriccell) [inline site/apache.md]: The generated yaml files will look similar except with  un-commented entries like bellow.

File: site/apache.md [hunk 9]
```diff
@@ -0,0 +1,460 @@
...
+
+```
+
+# Load balancer web port
+
+loadBalancerWebPort: 30305
+
+```
+
+The user can access an application from utsidie of the Kubernetes cluster via http://<host>:30305/<application-url>.
```
Comment (rosemarymarano) [inline site/apache.md]: typo utsidie, outside

File: site/apache.md (hunk 7 above)
Comment (rosemarymarano) [inline site/apache.md]: "Once the loadBalancerVolumePath is specified...", should be "After the loadBalancerVolumePath is specified..."

File: site/apache.md [hunk 10]
```diff
@@ -0,0 +1,460 @@
...
+
+        - name: "domain1-apache-webtier"
+
+          mountPath: "/config"
+
+```
+
+
+
+## When use Apache load balancer with a manually created WebLogic Domain
```
Comment (rosemarymarano) [inline site/apache.md]: Should this be "When to use the Apache..."

File: site/apache.md (hunk 1 above)
Comment (doxiao) [reply site/apache.md]: I used "prepare" since the steps include download, build and tag, not just build.

File: site/apache.md (hunk 10 above)
Comment (doxiao) [reply site/apache.md]: I am changing it to "Use the Apache...."

File: site/apache.md (hunk 6 above)
Comment (doxiao) [reply site/apache.md]: I am changing it to "Use the Apache...."

File: site/apache.md (hunk 1 above)
Comment (doxiao) [reply site/apache.md]: What about "You need to have the..."?

Comment (rosemarymarano) [reply site/apache.md]: Just noticed a typo here, enbeds, embeds.

Comment (mriccell) [reply site/apache.md]: prepare is fine

**Pull request merged**